public/plugin.ts

```diff
@@ -64,6 +64,10 @@ export class WazuhPlugin implements Plugin<WazuhSetup, WazuhStart, WazuhSetupPlu
   }
 
   public async start(core: CoreStart, plugins: AppPluginStartDependencies): Promise<WazuhStart> {
+    // hide security alert 
+    if(plugins.security_oss)
+      plugins.security_oss.insecureCluster.hideAlert(true);
```

Comment:
Thi's it only for cluster or it's generic?

Reply:
It is the only way to manage the events of this alert.